src/test/com/artos/testcase/TestCase_1.java

```diff
@@ -0,0 +1,19 @@
+package test.com.artos.testcase;
+
+import com.artos.annotation.TestCase;
+import com.artos.annotation.TestPlan;
+import com.artos.annotation.Unit;
+import com.artos.framework.infra.TestContext;
+import com.artos.interfaces.TestExecutable;
+
+@TestPlan(preparedBy = "Alan", preparationDate = "2019/11/6")
+@TestCase
+public class TestCase_1 implements TestExecutable {
+
+    @Unit
+    public void testUnit_1_1(TestContext context) {
+        // --------------------------------------------------------------------------------------------
+        System.out.println("testUnit_1_1");
+        // --------------------------------------------------------------------------------------------
+    }
+}
```

Comment:
What does this test achieves?

Reply:
I create TestCase_1 and TestCase_2 as dummy test cases for listing on the GUI selector, I see them as stubs for the real project test cases. They will print a message on console so we can see it is executed.